Resources/Prototypes/_Shitmed/Tiles/abductor.yml

```
@@ -2,9 +2,14 @@
   sprite: /Textures/_Shitmed/Tiles/abductor_tile.png
+  baseTurf: Lattice # Mono
   isSubfloor: false # Mono
+  # Mono - Fireaxe plating prying
+  deconstructTools: [ Axing ]
+  deconstructTimeMultiplier: 12.5 # Mono
   footstepSounds:
     collection: FootstepPlating # Mono
   heatCapacity: 10000
-  weather: false # Mono
-  indestructible: false # Mono
+ # weather: true # Mono
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)